File: postgraduate/src/main/webapp/js/service/StudentService.js
app.service("StudentService",function($http){
	//查询全部学生
	this.selectAll=function(){
		return $http.get('../Student/selectAll.do');
	}
	//单条查询
	this.selectOne=function(id){
		return $http.get('../Student/selectOne.do?id='+id);
	}
	//查询学生密码
	this.selectPassById=function(id){
		return $http.get('../Student/selectPassById.do?id='+id);
	}
	//分页
	this.findPage=function(page,rows){
		return $http.get('../Student/findPage.do?page='+page+'&rows='+rows);
	}
	//修改
	this.update=function(entity){
		return $http.post('../Student/update.do',entity);
	}
	//修改密码
	this.updatePassById=function(entity){
		return $http.post('../Student/updatePassById.do',entity);
	}
	//增加
	this.insert=function(entity){
		return $http.post('../Student/insert.do',entity);
	}
	//模糊查询分页
	this.search=function(page,rows,searchEntity){
		return $http.post('../Student/search.do?page='+page+'&rows='+rows,searchEntity);
	}
	
});

File: postgraduate/src/main/java/com/postgraduate/service/StudentService.java
package com.postgraduate.service;

import java.util.List;


import com.postgraduate.pojo.TbStudent;

import entity.PageResult;

public interface StudentService {
	/**
	 * 查询全部学生
	 * @return
	 */
	public List<TbStudent> selectAll();
	/**
	 * 查询单个学生
	 * @return
	 */
	public TbStudent selectOne(int id);
	/**
	 * 模糊查询学生
	 * @return
	 */
	public List<TbStudent> selectPart(TbStudent tbStudent);
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	/**
	 * 模糊查询返回分页列表
	 * @return
	 */
	public PageResult findPage(TbStudent tbStudent,int pageNum,int pageSize);
	/**
	 * 新增加学生
	 * @return
	 */
	public void insert(TbStudent tbStudent);
	/**
	 * 修改学生信息
	 * @return
	 */
	public void update(TbStudent tbStudent);
	/**
	 * 修改学生密码
	 * @return
	 */
	public void updatePassById(TbStudent tbStudent);
	

}
